### ISSUE TITLE
`.build(PostgresQueryBuilder)` does not construct valid SQL with `.is_null()`

### ISSUE THREAD

**Issue by ezracelli** [SeaQL/sea-query#31]
Using `sea_query` version `0.8` and `tokio_postgres` version `0.7`.

```rust
let (statement, params) = sea_query::Query::select()
    .columns(vec![
        iden::Model::Id,
    ])
    .from(iden::Model::Table)
    .and_where(
        sea_query::Expr::tbl(
            iden::Model::Table,
            iden::Model::DeletedAt,
        )
        .is_null(),
    )
    .build(sea_query::PostgresQueryBuilder);

println!("{} -- {:?}", statement, params);

let statement = conn.prepare(&*statement).await?;
let rows = conn.query(&statement, &*params.as_params()).await?;
```

Output:

```
SELECT "id" FROM "models" WHERE "models"."deleted_at" IS $1 -- Values([Null])
Error: db error: ERROR: syntax error at or near "$1"
```

Expected output:

```
SELECT "id" FROM "models" WHERE "models"."deleted_at" IS NULL -- Values([])
```

**Comment by tyt2y3** [comment]
Thank you for your report. Should have fixed in `0.8.4` via https://github.com/SeaQL/sea-query/commit/769356f91b8754081fdf6ed7ad12dc24e9343b8e

**Comment by ezracelli** [comment]
Wow, thanks for the incredibly quick turnaround! I had just forked the repo to write a PR when I noticed there was a new commit 😁 

`sea-query` is a great little project, and I'm working on migrating from my existing hand-written queries &mdash; I might raise a few more issues if/when they arise, but I'm more than happy to write PRs for them
